src/psyclone/psyir/tools/call_tree_utils.py

```diff
@@ -278,6 +278,12 @@ def get_non_local_read_write_info(self, node_list, read_write_info):
         '''Returns the information about non-local variables that are read
         or written.
 
+        :param node_list: list of nodes containing Kernel calls to interrogate.
```

Comment:
Gee, embarrassing - thanks for fixing the missing parameters.

Reply:
Well, I think I reviewed it so it's embarrassment shared :-)